.github/workflows/CI.yml

```diff
@@ -0,0 +1,65 @@
+name: CI
+
+on:
+  pull_request:
+    branches: [main]
+  push:
+    branches: [main]
+  workflow_dispatch:
+env:
+  PY_VERSION: "3.11"
+
+jobs:
+  changes:
+    runs-on: ubuntu-latest
+    # Required permissions
+    permissions:
+      pull-requests: read
+    # Set job outputs to values from filter step
+    outputs:
+      files: ${{ steps.filter.outputs.files }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4.1.7
+
+      - name: Filter files
+        uses: dorny/paths-filter@de90cc6fb38fc0963ad72b210f1f284cd68cea36 #v3.0.2
+        id: filter
+        with:
+          filters: |
+            files:
+              - 'setup.py'
+              - 'pyproject.toml'
+              - '.conda/env_build.yml'
+              - '.conda/meta.yml'
+
+  verify-conda-build:
+    name: Conda Build
+    runs-on: ubuntu-latest
+    needs: changes
+    # Only run if there are changes in the build files
+    if: ${{ needs.changes.outputs.files == 'true' }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup conda build environment
+        uses: conda-incubator/setup-miniconda@a4260408e20b96e80095f42ff7f1a15b27dd94ca # v3.0.4
+        with:
+          miniconda-version: "latest"
+          python-version: ${{ env.PY_VERSION }} 
+          environment-file: .conda/env_build.yml
+          auto-activate-base: false
+          auto-update-conda: false
+          show-channel-urls: true
+
+      - name: Verify conda recipe
+        shell: bash -el {0}
+        run: conda-verify .conda --ignore C2105,C2122
+
+      - name: Run conda build
+        shell: bash -el {0}
+        run: conda build . --no-anaconda-upload --output-folder=./build -c conda-forge -c accessnri -c coecms
+
+      - name: Verify conda package
+        shell: bash -el {0}
+        run: conda-verify ./build/noarch/*.tar.bz2 --ignore C1105,C1115,C1141
```

Comment:
Similarly here - could we have a comment in the code on what we are ignoring here?